content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,61 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+systemctl stop jenkins
+```
+2) Take Jenkins instance backup as a recovery option ::
+[source,bash]
+```
+#Take backup of Jenkins home directory. In my case my Jenkins home directory is /var/lib/jenkins.
+```
+
+See the link:/doc/book/system-administration/backing-up/[backup instructions] for alternatives.
+
+5) Start The Jenkins Service ::
```

Comment:
this goes from step 2 to step 5, I would advise using the [ordered list formatting](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/) so that the numbers are kept in order by the document itself.

Reply:
I have used the description list after experimenting with [ordered list formatting](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/) but the problem is ordered list formatting won't keep the order by itself if not written like this

```
. Protons
. Electrons
. Neutrons
```
writing some paragraphs between ordered list formatting won't work, we can make do use _start_ but it isn't recommended, [see here](https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/)